docs/visual-basic/language-reference/operators/mod-operator.md

```diff
@@ -38,12 +35,34 @@ number1 Mod number2
  `number2`  
  Required. Any numeric expression.  
   
-## Supported Types  
+## Supported types  
  All numeric types. This includes the unsigned and floating-point types and `Decimal`.  
   
-## Result  
- The result is the remainder after `number1` is divided by `number2`. For example, the expression `14 Mod 4` evaluates to 2.  
-  
+## Result
+
+The result is the remainder after `number1` is divided by `number2`. For example, the expression `14 Mod 4` evaluates to 2.  
+
+> [!NOTE]
+> There is a difference between *remainder* and *modulus* in mathematics, with different results for negative numbers. The `Mod` operator in Visual Basic, the .NET Framework op_Modulus operator, and the underlying [rem]<xref:System.Reflection.Emit.OpCodes.Rem> IL instruction all perform a remainder operation.
+
+The result of a `Mod` operation is always in the range (`number2`, -`number2`), exclusive, and retains the sign of the dividend, (`number1`). For example:
```

Comment:
Could we say this more simply as (assuming we use the terms dividend and divisor frequently in the docs). 

The result of a `Mod` operation retains the sign of the dividend, (`number`1), so may be positive or negative. If the dividend is positive, the result is always between zero and the divisor (`number 2`). If the dividend is negative, the result is always between zero and the negative of the divisor (`-number 2`). For example:

Reply:
The range (`number2`, `-number2`) reads unusual for negative numbers, e.g., (3, -3). I like the version suggested by @KathleenDollard but it's doesn't work when `number2` is negative. What about the following edition (I might be wrong with the `Abs` syntax):

The result of a `Mod` operation retains the sign of the dividend, (`number1`), so may be positive or negative. If the dividend is positive, the result is always between zero and the absolute value of the divisor (`Abs(number2)`). If the dividend is negative, the result is always between the negative of the absolute value of the divisor (`-Abs(number2)`) and zero. For example:

Reply:
OR:

The result of a `Mod` operation retains the sign of the dividend, `number1`, so may be positive or negative. The absolute value of the result is less than the absolute value of the divisor, `Abs(number2)`

Reply:
Another way is:

The result of a Mod operation retains the sign of the dividend, `number1`, so may be positive or negative. The result is always between 0 and `number2`.

Reply:
Consider number1 = -8, number2 = 3. Then result is -2 and not between 0 and 3. I like that @rpetrusha shows all possible sign combinations in the example. Maybe it's better than all the words :)

Reply:
I like your correction to mine, but agree that the point is to support the great examples. I wonder if calling out ignoring the divisor makes sense. 

The result of a Mod operation retains the sign of the dividend, (number1), so may be positive or negative. The sign of the divisor is ignored and the absolute value is used, (Abs(number2)).  If the dividend is positive, the result is always between zero and the absolute value of the divisor (Abs(number2)). If the dividend is negative, the result is always between the negative of the absolute value of the divisor (-Abs(number2)) and zero. For example:

I'm fine with any of them, but found I had to do mental gymnastics to figure out the range explanation, even if it is more concise. If we can be verbose here, we could include both as they will speak to different people.